parser/src/string.rs

```diff
@@ -191,7 +183,7 @@ impl<'a> StringParser<'a> {
         let mut conversion = ConversionFlag::None;
         let mut self_documenting = false;
         let mut trailing_seq = String::new();
-        let location = self.get_pos();
+        let location = self.get_pos() - TextSize::from(1);
```

Comment:
Is it guarnanteed that the character one position before is a ASCII character? I think it is because it is always the `}` token. Would it make sense to instead pass the start location as an argument?

Reply:
thanks for prodding me on these. think i have a better solution (though it does add a new dependency. happy to discuss)